src/common/set_up_engine.m

```diff
@@ -0,0 +1,46 @@
+function alias = set_up_engine(engine)
+% Creates a string, evaluating which imports PET engine.
```

Comment:
this docstring doesn't make a lot of sense to me. When there's no output-arg, there's no string being created. And when there is, `alias` is a `struct`.

Also, it refers to PET.

Best to include an example in the docstring. also add
```matlab
% See also set_up_PET and set_up_MR
```
(see https://www.mathworks.com/help/matlab/matlab_prog/add-help-for-your-program.html)

similar changes required in the docstring of `set_up_PET` etc